portal/src/components/Typography/FormatProvider.tsx

```diff
@@ -26,9 +26,9 @@ const components = {
     h4: HeadingSmall,
     h5: HeadingXS,
     p: Paragraph,
-    ul: UnorderedList,
-    ol: OrderedList,
-    li: ListItem,
+    ul: UnorderedList as React.FC,
+    ol: OrderedList as React.FC,
+    li: ListItem as React.FC,
```

Comment:
Kan man ikke gjøre dette i selve komponenten? 🤔

Reply:
Det er den breaking changen, den kommer ut med `React.FC<Props>`, men her caster vi den til `React.FC<{}>` for å få den til å leke med FormatProvider komponenten.